apps/bond/components/BondingProducts/Bonding/Deposit/Deposit.jsx

```diff
@@ -5,12 +5,9 @@ import { isNil } from 'lodash';
 import PropTypes from 'prop-types';
 import { useCallback, useEffect, useState } from 'react';
 
-import {
-  COLOR,
-  getCommaSeparatedNumber,
-  notifyError,
-  notifySuccess,
-} from '@autonolas/frontend-library';
+import { getCommaSeparatedNumber, notifyError, notifySuccess } from '@autonolas/frontend-library';
```

Comment:
can we import from `libs/util-functions/src`? at least notifyError and notifySuccess